Dependency/peg-markdown-highlight/HGMarkdownHighlighter.m

```diff
@@ -258,7 +258,9 @@ - (void) clearHighlightingForRange:(NSRange)range
 	[textStorage applyFontTraits:_clearFontTraitMask range:range];
 	[textStorage removeAttribute:NSBackgroundColorAttributeName range:range];
 	[textStorage removeAttribute:NSLinkAttributeName range:range];
-    if (self.targetTextView.typingAttributes && self.resetTypingAttributes)
+    if (self.targetTextView.typingAttributes
+        && self.resetTypingAttributes
+        && self.defaultTypingAttributes[NSParagraphStyleAttributeName] != NULL)
```

Comment:
Thanks for catching that! My Obj-C is getting a bit rusty with all the full-time Swift :)